### PR TITLE
MMCA 5042 - Implement scalafmt plugin on services V3

### PR DESCRIPTION
**Pull request by jamie2210**
- [x] Add .scalafmt.conf
- [x] Add plugins
- [x] Add scalafmtCheckAll and runAllChecks
- [x] Pass conflicts with scalastyle

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+project {
+ git = true
+}
+
+
+align {
+ preset = most
+ tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "=", "<-", "->", "%", "%%", "should", "shouldBe", "shouldEqual", "shouldNot", "must" ]
```
Comment (AdamCoops) [inline .scalafmt.conf:15]: can we break this into two lines due to length?

Comment (jamie2210) [reply .scalafmt.conf]: no probs, that's done.

File: .scalafmt.conf [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+runner.dialect = scala3
+maxColumn = 120
+lineEndings = unix
+importSelectors = binPack
+
+
+project {
+ git = true
+}
+
```
Comment (AdamCoops) [inline .scalafmt.conf:11]: Do we need two empty spaces between each section on this page?

Comment (jamie2210) [reply .scalafmt.conf:11]: Removed.

**Review by AdamCoops — APPROVED** [review]

**Pull request merged**